installers/linux/universal/deb/build.gradle

```diff
@@ -20,7 +20,7 @@
  */
 
 plugins {
-    id 'nebula.ospackage' version '8.4.1'
+    id 'nebula.ospackage' version '9.0.0'
```

Comment:
Can we use dynamic plugin versioning? https://docs.gradle.org/7.0/release-notes.html#using-dynamic-versions-in-the-plugins-block